cosmo_tester/conf_cli.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+# While the config is yaml, using json to dump the examples of it gives a
+# cleaner output
+import json
+
+from cosmo_tester.framework.config import (
+    load_config,
+    validate_config,
+)
+from cosmo_tester.framework.logger import get_logger
+
+
+def show_schema(schema,
+                generate_sample_config=False,
+                include_defaults=False,
+                indent=''):
+    output = []
+    sorted_config_entries = schema.keys()
+    sorted_config_entries = [
+        entry for entry in sorted_config_entries
+        if isinstance(schema[entry], dict)
```

Comment:
namespace: true

Reply:
hmm, no, I think it's still a dict
```
self.schema[namespace] = {'.is_namespace': True}
```

Reply:
That's set on the in-memory copy when it's loaded, so that an appropriate alert can be provided. The actual key:value in the schema itself to define that something is a namespace is: namespace: true

Reply:
Side note: I was wrong, it's for removing the .is_namespace: True entries from the sorted list.